client/messaging/common-message-layout.tsx

```diff
@@ -212,14 +214,19 @@ export const BlockedMessage = React.memo<{
   time: number
   text: string
 }>(props => {
+  const { t } = useTranslation()
   const [show, setShow] = useState(false)
 
   return (
     <>
       <TimestampMessageLayout time={props.time} highlighted={false}>
-        <BlockedText>Blocked message</BlockedText>
+        <BlockedText>{t('messaging.blockedMessage', 'Blocked message')}</BlockedText>
         <BlockedDivider>&mdash;</BlockedDivider>
-        <ShowHideLink onClick={() => setShow(!show)}>{show ? 'Hide' : 'Show'}</ShowHideLink>
+        <ShowHideLink onClick={() => setShow(!show)}>
+          {show
+            ? t('messaging.hideBlockedMessage', 'Hide')
```

Comment:
Can probably throw these strings under `common.actions.hide` and `common.actions.show`